edit_course/views.py

```diff
@@ -320,6 +320,8 @@ def get_form_kwargs(self):
 
     def form_valid(self, form):
         from .operations.clone import clone
+        if not 'sis' in form.cleaned_data:
+            form.cleaned_data['sis'] = 'none'
```

Comment:
If the `clone()` function accepts empty `siskey` values too, then these lines are not needed.
```suggestion
```